src/uu/cp/src/cp.rs

```diff
@@ -353,9 +354,22 @@ fn show_debug(copy_debug: &CopyDebug) {
 const ABOUT: &str = help_about!("cp.md");
 const USAGE: &str = help_usage!("cp.md");
 const AFTER_HELP: &str = help_section!("after help", "cp.md");
+// Considering 4 Kb as block size for a standard file system, multiply it by 4 to make it 16 kb
+const BUFFER_SIZE: usize = 4096 * 4;
+const BUFFER_SIZE_U64: u64 = BUFFER_SIZE as u64;
 
 static EXIT_ERR: i32 = 1;
 
+fn get_size_limit_for_normal_copy() -> u64 {
+    const DEFAULT_SIZE_LIMIT_FOR_NORMAL_COPY: u64 = 524288000; // 500 Mb
+    match std::env::var("CP_SIZE_LIMIT_FOR_NORMAL_COPY") {
+        Ok(val) => val
+            .parse::<u64>()
+            .unwrap_or(DEFAULT_SIZE_LIMIT_FOR_NORMAL_COPY),
+        Err(_) => DEFAULT_SIZE_LIMIT_FOR_NORMAL_COPY,
+    }
+}
+
```

Comment:
can probably be simplified with:
```suggestion
fn get_size_limit_for_normal_copy() -> u64 {
    const DEFAULT: u64 = 524_288_000; // 500 Mb
    std::env::var("CP_SIZE_LIMIT_FOR_NORMAL_COPY")
        .and_then(|val| val.parse().ok())
        .unwrap_or(DEFAULT)
}

```

Reply:
Done! Looks more concise now. Thanks!